pkg/reconcile/reconcile.go

```diff
@@ -91,3 +93,25 @@ var _ Reconciler = Func(nil)
 
 // Reconcile implements Reconciler.
 func (r Func) Reconcile(o Request) (Result, error) { return r(o) }
+
+// PanicHandler can be called with defer in order to recover from panic in reconcile logic.
+// Additional handlers can also be passed to manage the go routines. The error passed to the
+// PanicHandler is set to the value of panic if not nil.
+// If any resource triggers a panic, the reconciler will recover and modify the raised panic
+// to the normal reconciler error.
+// Example usage:
+/*
+func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (result reconcile.Result, err error) {
+	defer HandleCrashReconciler(&err)
```

Comment:
What's `HandleCrashReconciler()` over here?
Is this supposed to be `defer reconcile.PanicHandler(&err)`?

Reply:
Ops, Im sorry, that's a mistake. Yes, that's supposed to be `reconcile.PanicHandler(&err)`. Will change that.